en/docs/services/service-concepts.md

```diff
@@ -0,0 +1,113 @@
+   |                             | reopened               | Reopen a pull request                    |
+   | pull_request_review         | submitted              | Submit a pull request                    |
+   |                             | edited                 | Edit a pull request review               |
+   |                             | dismissed              | Dismiss a pull request review            |
+   | pull_request_review_comment | created                | Create a pull request review comment     |
+   |                             | edited                 | Edit a pull request review comment       |
+   |                             | deleted                | Delete a pull request review comment     | 
+
+
+
+## Connection
+
+A connection in Choreo allows you to connect to and interact with an external service through your application. You can use a connection to interact with generic protocol-based APIs and application specific APIs.
+
+Following are the connections you can use to interact with generic protocol-based APIs:
+  
+ - **HTTP**: Allows your application to interact with the HTTP API and communicate with an external endpoint via the HTTP protocol.   
+ - **SMTP**: Allows your application to send emails via the SMTP protocol.   
+ - **POP3**: Allows your application to receive emails from a POP3 email server.  
+ - **IMAP**: Allows your application to receive emails from any email server that supports the IMAP protocol. 
+
+Following are the connections you can use to interact with application specific APIs:
+
+ - **GitHub**: Allows your application to interact with the GitHub API and  perform required operations. 
+ - **Gmail**: Allows your application to interact with the Gmail API and perform operations such as creating, modifying, and sending emails.
+ - **Google Calendar**: Allows your application to interact with the Google Calendar API and perform required operations. 
+ - **Google Sheets**: Allows your application to interact with the Google Sheets API and perform operations such as creating and listing spreadsheets.
+ - **Twilio**: Allows your application to interact with the Twilio API and perform operations such as sending messages, WhatsApp messages, and making voice calls.
+
+
+
+## Statement
+
+A statement is a syntactic unit that specifies the action to be carried out in a Choreo application. 
+
+Choreo supports the following statements:
+
+ - **Log**: Allows you to add an entry to the application log at runtime. You can either add an **Info** log or an **Error** log depending on your requirement   
+ - **If**: Allows you to execute a collection of statements if a specified condition is met.    
+ - **Variable**: Allows you to declare a variable of a preferred type depending on your requirement.   
+ - **For-each**: Allows you to traverse through items in a collection of data. For example, traverse through arrays, maps, JSON, XML, and tables.  
```

Comment:
Is it ok to say the explicit types such as tables here as we don't have tables support in choreo?

Reply:
Removed "For example, traverse through arrays, maps, JSON, XML, and tables." for the moment. Need to check on the exact support and include if necessary.